apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -620,6 +758,33 @@ export const FileBrowser = React.memo(() => {
     [project.folders, dialogControl, renameFolder, appName]
   );
 
+  const handleDragEnd = useMemo(
+    () => (e: DragOverEvent) => {
+      if (e?.over) {
+        if (e.active.data.current?.type === 'FOLDER') {
```

Comment:
ChatGPT suggests

```
export const DRAG_TYPE_FILE = 'FILE';
export const DRAG_TYPE_FOLDER = 'FOLDER';
```

or, "if you prefer a more concise style" 😅 
```
export const FILE_TYPE = 'FILE';
export const FOLDER_TYPE = 'FOLDER';
```

Reply:
I opted for an enum called `DragType`, with values for `FILE` and `FOLDER` instead.

Since these are constants that are _only_ used in the DnD implementation.

Reply:
Are you still planning on adding the constants in this PR? I don't see them yet.

Reply:
I thought I _had_ added those constants. I have no idea what happened to that commit. Bizarre.

Anyway, actually added in now.